components/schemes/Color/src/MDCSemanticColorScheme.m

```diff
@@ -22,6 +22,10 @@
                           blue:(CGFloat)((colorValue & 0xFF) / 255.0) alpha:1];
 }
 
+static double blendColorChannel(double value, double bValue, double alpha, double bAlpha) {
```

Comment:
Please add documentation for parameters.

Reply:
Done.